src/wallet/test/wallet_tests.cpp

```diff
@@ -29,7 +29,7 @@ extern UniValue importmulti(const JSONRPCRequest& request);
 
 std::vector<std::unique_ptr<CWalletTx>> wtxn;
 
-typedef std::set<std::pair<const CWalletTx*,unsigned int> > CoinSet;
+typedef std::set<CInputCoin> CoinSet;
```

Comment:
We don't use that style anywhere, introducing it here is not a good idea. I'd prefer to save that for new code.